ormolu.cabal

```
@@ -32,6 +32,10 @@ flag dev
     default:     False
     manual:      True
 
+flag internal-bundle-fixities
+    description: INTERNAL FLAG DONT USE
```

```suggestion
    description: An internal ad-hoc flag that is enabled by default, Ormolu live disables it
```

I do not know why, perhaps due to the size of the byte string that would be embedded? Feel free to add a better explanation.

---

See https://github.com/tweag/ormolu/pull/964#discussion_r1085635501